src/Client/Suggest.h

```diff
@@ -28,8 +28,8 @@ class Suggest : public LineReader::Suggest, boost::noncopyable
     template <typename ConnectionType>
     void load(ContextPtr context, const ConnectionParameters & connection_parameters, Int32 suggestion_limit);
 
-    /// Older server versions cannot execute the query above.
-    static constexpr int MIN_SERVER_REVISION = 54406;
+    /// Older server versions cannot execute the query loading suggestions.
+    static constexpr int MIN_SERVER_REVISION = DBMS_MIN_PROTOCOL_VERSION_WITH_VIEW_IF_PERMITTED;
```

Comment:
Does this break existing old proton client ?

Reply:
No, this won't break anything.

However, this PR will break the existing old proton client, that's because, again, the old client is using revision of 54459, but it skips a few revision changes. I don't think this is a big issue today, because we don't provide standalone proton client downloads, I don't think anyone need to connect a proton server with a different proton client.

Reply:
This is exactly my ask. Does it break existing old proton client. Seems yes. Let's give a head up to Jove

Reply:
Thanks for the heads up. Most of the case, I launch `proton client` from the docker container to connect to the local proton. However if someone downloads the single binary proton via curl or homebrew, and use that as a client to connect to newer version of proton as docker, then this could break. But I don't think it's a concerning issue.